Assets/LeapMotionModules/Interaction/InteractionHand.cs

```diff
@@ -218,8 +229,7 @@ private struct HoverCheckResults {
 
     private void FixedUpdateTouch() {
       _touchActivityManager.activationRadius = interactionManager.WorldTouchActivationRadius;
-
-      _touchActivityManager.FixedUpdateHand(_hand);
+      _touchActivityManager.FixedUpdateHand((_hand != null) ? _hand.PalmPosition.ToVector3() : Vector3.zero);
```

Comment:
sounds like we should create some sort of method in the activity manager called ClearHand or ClearActive or something.  All it needs to do at that point is clear the active list right?